changelog.md

```diff
@@ -57,7 +56,7 @@
 ### Operations
 
 * Use dh-virtualenv and mod_wsgi to create securedrop-app-code package, run Python 3 version of the web applications on instances (#4622).
-* Adds support for v3 onion services for SecureDrop source, journalist, and SSH interfaces (#4652, #4710, #4690, #4675).
+* Adds support for v3 onion services for SecureDrop source, journalist, and SSH interfaces (#4652, #4710, #4690, #4675, #4844).
```

Comment:
hmm, the thing with these historical changelog edits is that we've already cut the 1.0.0 tag - let's just remove from the upcoming 1.1.0 release notes

Reply:
How about adding a 1.0.0 (Addendum) section above? Mostly concerned that #4838 in particular was a pretty hefty change (including to the threat model) and I want to make sure folks can find it if they look for it here.

Reply:
yeah i'm ok with this